src/node.rs

```diff
@@ -1440,6 +1432,9 @@ impl<S: Send + Sync + 'static + ForkSource + std::fmt::Debug> EthNamespaceT for
                             zksync_types::l2::TransactionType::EIP1559Transaction => 2,
                             zksync_types::l2::TransactionType::EIP712Transaction => 113,
                             zksync_types::l2::TransactionType::PriorityOpTransaction => 255,
+                            zksync_types::l2::TransactionType::ProtocolUpgradeTransaction => {
+                                todo!()
```

Comment:
Please use `not_implemented("...")` so we can be consistent about console output for unimplemented features of `era_test_node`